sktalk/corpus/parsing/cha.py

```diff
@@ -5,32 +5,67 @@
 
 
 class ChaFile(InputFile):
+    TIMING_REGEX = r"\D(?P<timing>\d{1,9}_\d{1,9})\D"
+    # participant may not be repeated on each line
+    PARTICIPANT_REGEX = r"(^\*(?P<participant>[^:*]+)\:){0,1}"
```

Comment:
Unless the participant name can also be terminated by `*` instead of `:`, the * in the `[^:*]` group seems redundant. Can it? I thought it always had to have the form `*<NAME>:` where `<NAME>` must not contain a `: `.

```suggestion
    PARTICIPANT_REGEX = r"(^\*(?P<participant>[^:]+)\:){0,1}"
```
On the other hand, if the participant can be specified in the form `*NAME*`, there should also be a test case added below.

I might not fully understand the comment in the line above, though. Is this to exclude a potential second participant name in the same line?